airbyte-integrations/connectors/destination-mongodb/src/main/java/io/airbyte/integrations/destination/mongodb/MongodbDestinationStrictEncrypt.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.mongodb;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.db.mongodb.MongoUtils.MongoInstanceType;
+import io.airbyte.integrations.base.Destination;
+import io.airbyte.integrations.base.IntegrationRunner;
+import io.airbyte.integrations.base.spec_modification.SpecModifyingDestination;
+import io.airbyte.protocol.models.AirbyteConnectionStatus;
+import io.airbyte.protocol.models.AirbyteConnectionStatus.Status;
+import io.airbyte.protocol.models.ConnectorSpecification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Currently this is a duplicate of the same file within destination-mongodb-strict-encrypt module.
```

Comment:
@tuliren just wanted to check, is there anything stopping me from moving the `MongodbDestinationStrictEncrypt` file from the `destination-mongodb-strict-encrypt` module instead of creating a copy of the `MongodbDestinationStrictEncrypt` in the non-strict-encrypt variant?

Currently hesitated to move the `strict-encrypt` variant since a duplicate file exists for the `source-postgres` and `source-postgres-strict-encrypt`

Reply:
> moving the `MongodbDestinationStrictEncrypt` file from the `destination-mongodb-strict-encrypt` module

After moving the strict encrypt class from the `strict-encrypt` module to the normal module, will the `strict-encrypt` module still have a `mainClass` in its `build.gradle` file?  If yes, this would be the perfect solution for now. We won't have the duplicated class problem.